actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -454,12 +454,16 @@ def initialize(parent_jar, key_generator, options = {})
         @verifier = ActiveSupport::MessageVerifier.new(secret, digest: digest, serializer: ActiveSupport::MessageEncryptor::NullSerializer)
       end
 
+      # Returns the value of the cookie by +name+ if it is untampered,
+      # returns +nil+ otherwise and if no such cookie exists.
```

Comment:
This makes it sound like nil is only returned if the cookie has been tampered with AND the cookie does not exist. Is that technically correct? Did you mean "or" instead of "and"?